scripts/ci/dockerfiles/krb5-kdc-server/Dockerfile

```diff
@@ -61,11 +63,11 @@ COPY supervisord.conf /etc/supervisord.conf
 COPY entrypoint.sh /entrypoint.sh
 ENTRYPOINT ["/entrypoint.sh"]
 
-LABEL org.apache.airflow.component="krb5-kdc-server"
-LABEL org.apache.airflow.krb5-kdc-server.core.version="krb5"
-LABEL org.apache.airflow.airflow_bats.version="${AIRFLOW_KRB5KDCSERVER_VERSION}"
-LABEL org.apache.airflow.commit_sha="${COMMIT_SHA}"
-LABEL maintainer="Apache Airflow Community <dev@airflow.apache.org>"
+LABEL org.apache.airflow.component="krb5-kdc-server"\
```

Comment:
This is not detected by Hadolint, but I noticed it so I corrected it by the way. I am already working on submitting a request for a new feature for hadolint.